graphics/managed_surface.h

```diff
@@ -341,11 +340,10 @@ class ManagedSurface {
 	 *						the source surface
 	 * @param srcAlpha		Optional additional transparency applied to src
 	 * @param mask			Optional parameter with mask definition (0-skip, other-copy)
-	 * @param maskOnly		Optional parameter for using mask over transColor
 	 */
 	void transBlitFrom(const Surface &src, const Common::Rect &srcRect, const Common::Rect &destRect,
 		uint transColor = 0, bool flipped = false, uint overrideColor = 0, uint srcAlpha = 0xff,
-		const Surface *mask = nullptr, bool maskOnly = false);
```

Comment:
Could you please explain your rationale behind removing this flag?

Reply:
My original understanding of the flag was that it should always be true when the mask is supplied, and false otherwise. Thinking about it more, though, there may be be some use cases where the transparent colour should be used as well as the mask, or neither the transparent colour nor the mask should be used. If that's a useful thing to have, I can bring the flag back, but it should probably be renamed to better reflect it's purpose since it affects the use transparent colour, and whether or not there's a mask isn't that relevant.

Reply:
Yes, this was exactly the purpose: to have cheaper masked blits when I know for sure, the transparency is not present. Or, allow mixing-in alpha when a dev knows what he is doing.